lib/rsk-utils.js

```diff
@@ -333,20 +333,22 @@ const triggerRelease = async (rskTransactionHelpers, btcClient, callbacks = {})
  * Executes a method 'call' and calls the callback with the result of the call, then calls 'send' and waits for the transaction receipt to be available and returns it
  * @param {RskTransactionHelper} rskTxHelper to make transactions to the rsk network
  * @param {web3.eth.Contract.ContractSendMethod} method contract method to be invoked
- * @param {function} checkCallback callback to check the result of the method 'call' before calling 'send'
  * @param {string} from rsk address to send the transaction from
+ * @param {function} checkCallback callback to check the result of the method 'call' before calling 'send'
+ * @param {object} options optional. Object with the options for the transaction. It can contain the following properties: value, gasPrice, gas
  * @returns {web3.eth.TransactionReceipt} txReceipt
  */
-const sendTxWithCheck = async (rskTxHelper, method, from, checkCallback) => {
-
-  const callResult = await method.call({ from });
+const sendTxWithCheck = async (rskTxHelper, method, from, checkCallback, options = { value: 0, gasPrice: 0, gas: 0 }) => {
```

Comment:
Maybe receive options param first and checkCallback last? Mainly to avoid having to pass `null` when calling from `sendTransaction`

Reply:
Updated.